docker/backend/init.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
@ABrain7710 : do we support this way of loading anymore, or does it need to be depricated?